src/sd_jwt/bin/generate.py

```diff
@@ -39,7 +39,7 @@ def generate_test_case_data(settings: Dict, testcase_path: Path, type: str):
     use_decoys = testcase.get("add_decoy_claims", False)
     serialization_format = testcase.get("serialization_format", "compact")
     include_default_claims = testcase.get("include_default_claims", True)
-    extra_header_parameters = testcase.get("extra_header_parameters", None)
+    extra_header_parameters = testcase.get("extra_header_parameters", {"typ": "sd+jwt"})
```

Comment:
I think we should use "example+sd-jwt" as default which can be overridden if required.
Note that `sd+jwt` has not been defined anywhere, so we should not use it.

```suggestion
    extra_header_parameters = testcase.get("extra_header_parameters")
```